Applications/ConsoleReferenceClient/UAClient.cs

```diff
@@ -178,11 +178,7 @@ public async Task<bool> ConnectAsync(string serverUrl, bool useSecurity = true,
                     EndpointConfiguration endpointConfiguration = EndpointConfiguration.Create(m_configuration);
                     ConfiguredEndpoint endpoint = new ConfiguredEndpoint(null, endpointDescription, endpointConfiguration);
 
-#if NET6_0_OR_GREATER
-                    var sessionFactory = TraceableSessionFactory.Instance;
-#else
                     var sessionFactory = DefaultSessionFactory.Instance;
```

Comment:
there is still a keep alive issue if traceable session is used. will be fixed in another PR